ga4gh/datamodel/reads.py

```diff
@@ -225,13 +224,14 @@ def getSamFilePath(self):
         """
         return self._samFilePath
 
-    def getReadAlignments(self, referenceName=None, referenceId=None,
+    def getReadAlignments(self, referenceId=None,
                           start=None, end=None):
         """
         Returns an iterator over the specified reads
         """
-        if referenceName is not None and referenceId is not None:
-            raise exceptions.BadReadsSearchRequestBothRefs()
+        # TODO If referenceId is None, return against all references,
+        # including unmapped reads.
```

Comment:
@jeromekelleher I'm not sure I understand what you're asking - unless we're to re-introduce referenceName as an argument to this method (which we could, if you think it's a good idea), I don't believe there's a TODO to add for it: If referenceId is not none, the reference name is automatically looked up within the body of the method.